Tests/TuistGeneratorIntegrationTests/Utils/FrameworkMetadataProviderIntegrationTests.swift

```diff
@@ -0,0 +1,57 @@
+import Basic
+import Foundation
+import TuistCore
+import XCTest
+
+@testable import TuistCoreTesting
+@testable import TuistGenerator
+
+final class FrameworkMetadataProviderIntegrationTests: TuistTestCase {
+    var subject: FrameworkMetadataProvider!
+
+    override func setUp() {
+        super.setUp()
+        subject = FrameworkMetadataProvider()
+    }
+
+    override func tearDown() {
+        subject = nil
+        super.tearDown()
+    }
+
+    func test_bcsymbolmapPaths() throws {
+        // Given
+        let carthagePath = temporaryFixture("Carthage/")
+        let frameworkPath = FileHandler.shared.glob(carthagePath, glob: "*.framework").first!
+        let framework = FrameworkNode(path: frameworkPath)
+
+        // When
+        let got = try subject.bcsymbolmapPaths(framework: framework).sorted()
+
+        // Then
+        XCTAssertEqual(got.count, 2)
```

Comment:
super minor: as a suggestion, this could be written as an array comparison which should cover both the count and the contents.

e.g.

```swift
 XCTAssertEqual(got, [
     carthagePath.appending(component: "2510FE01-4D40-3956-BB71-857D3B2D9E73.bcsymbolmap"),
     carthagePath.appending(component: "773847A9-0D05-35AF-9865-94A9A670080B.bcsymbolmap")
])
```